File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        ds= {}
        dt = {}
        for i in s:
            if ds.get(i):
                ds[i]+=1
            else:
                ds[i]=1
        for i in t:
            if dt.get(i):
                dt[i]+=1
            else:
                dt[i]=1
        c = [0 for i in dt|ds if dt.get(i)!=ds.get(i)]
        return not len(c)

File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-1.cpp
class Solution {
public:
    int evalRPN(vector<string>& tokens) {
        stack<int> s;
        for(string &st:tokens){
            if (st == "+"){
                int v1 = s.top();
                s.pop();
                int v2 = s.top();
                s.pop();
                s.push(v1+v2);
            }
            else if (st == "-"){
                int v1 = s.top();
                s.pop();
                int v2 = s.top();
                s.pop();
                s.push(v2-v1);
            }
            else if (st == "*"){
                int v1 = s.top();
                s.pop();
                int v2 = s.top();
                s.pop();
                s.push(v1*v2);
            }
            else if (st == "/"){
                int v1 = s.top();
                s.pop();
                int v2 = s.top();
                s.pop();
                s.push(v2/v1);
            }
            else {
                s.push(stoi(st));
            }
        }
    return s.top();
    }
};
